### PR TITLE
feat/IPROD-93 Disabled ConnectionWizard page

### PR DESCRIPTION
**Pull request by geka-evk**
feat/IPROD-93 Disabled ConnectionWizard page

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Layout from './Layout';
 // import Dashboard from './Dashboard';
 import TechnicalDashboard from './TechnicalDashboard';
 import Transfers from './Transfers';
-import ConnectionWizard from './ConnectionWizard';
+// import ConnectionWizard from './ConnectionWizard';
```
Comment (mdebarros) [inline src/App/App.tsx:8]: If you comment out code, please put a comment indicating the reason.

File: src/App/App.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const App: FC<AppProps> = ({
                 <Route path="/dashboard" component={Dashboard} />
               */}
               <Route path="/transfers" component={Transfers} />
-              <Route path="/connections" component={ConnectionWizard} />
+              {/* <Route path="/connections" component={ConnectionWizard} /> */}
```
Comment (mdebarros) [inline src/App/App.tsx:50]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environment/Router.tsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ const EnvironmentPhasesTabs: FC<EnvironmentPhasesTabsProps> = ({
   onPhaseClick,
 }) => {
   const params = useParams<PhaseParams>();
-  const history = useHistory();
+  // const history = useHistory();
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environment/Router.tsx:77]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environment/Router.tsx [hunk 4]
```diff
@@ -126,7 +126,7 @@ const EnvironmentPhasesTabs: FC<EnvironmentPhasesTabsProps> = ({
             href="#null"
             onClick={(e) => {
               e.preventDefault();
-              history.push('/connections');
+              // history.push('/connections');
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environment/Router.tsx:129]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environments/Environments.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React, { FC } from 'react';
 import { AnimateFadeIn, Accordion, AccordionItem } from 'components';
-import { useHistory } from 'react-router-dom';
+// import { useHistory } from 'react-router-dom';
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environments/Environments.tsx:3]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environments/Environments.tsx [hunk 6]
```diff
@@ -17,7 +17,7 @@ const EnvironmentsOverview: FC<EnvironmentsOverviewProps> = ({
   environmentsStatuses,
   environmentsStatusesError,
 }) => {
-  const history = useHistory();
+  // const history = useHistory();
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environments/Environments.tsx:20]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environments/Environments.tsx [hunk 7]
```diff
@@ -41,7 +41,8 @@ const EnvironmentsOverview: FC<EnvironmentsOverviewProps> = ({
             status={status}
             statusColor={statusColor}
             progress={progress}
-            onClick={() => history.push(`/connections/${environment.name}`)}
+            onClick={() => {}}
+            // onClick={() => history.push(`/connections/${environment.name}`)}
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environments/Environments.tsx:45]: If you comment out code, please put a comment indicating the reason.

File: src/App/ConnectionWizard/Environments/Environments.tsx [hunk 8]
```diff
@@ -50,7 +51,7 @@ const EnvironmentsOverview: FC<EnvironmentsOverviewProps> = ({
                 isEnvironmentsStatusesPending
               );
 
-              const onClick = () => history.push(`/connections/${environment.name}/${phase.param}`);
+              // const onClick = () => history.push(`/connections/${environment.name}/${phase.param}`);
```
Comment (mdebarros) [inline src/App/ConnectionWizard/Environments/Environments.tsx:54]: If you comment out code, please put a comment indicating the reason.

File: src/App/Layout/SideMenu/index.tsx [hunk 9]
```diff
@@ -33,7 +33,7 @@ const SideMenu: FC<SideMenuProps> = ({ pathname }) => {
         </MenuSection>
         <MenuSection label="TECHNICAL OPS">
           <MenuItem path="/techdashboard" label="Overview" />
-          <MenuItem path="/connections" partial label="Connection Wizard" />
+          {/* <MenuItem path="/connections" partial label="Connection Wizard" /> */}
```
Comment (mdebarros) [inline src/App/Layout/SideMenu/index.tsx:36]: If you comment out code, please put a comment indicating the reason.

File: src/App/App.tsx (hunk 1 above)
Comment (geka-evk) [reply src/App/App.tsx]: i've removed commented out code

**Review by mdebarros — APPROVED** [review]
+1

**Comment by kalinkrustev** [comment]
merged in #26

**Pull request closed without merging**